### DISCUSSION TITLE
Velero support s3/gcs metrics?

### DISCUSSION THREAD

**Discussion by z2665** [vmware-tanzu/velero#7873] [Community support Q&A]
We currently use velero plus velero-plugin-for-gcp and velero-plugin-for-gcp to back up our cluster data to object storage in the cloud.
In order to better track the cost of the object storage used by velero, we would like to have metrics that capture the number of times the object storage api is accessed and the amount of data uploaded during the velero backup/recovery process, is there a metrics that velero can support?

**Comment by Lyndon-Li** [comment]
Not at present, created an issue for the same.
